eng/packaging.targets

```diff
@@ -5,10 +5,7 @@
     <!-- Don't restore prebuilt packages during sourcebuild. -->
     <DisablePackageBaselineValidation Condition="'$(DotNetBuildFromSource)' == 'true'">true</DisablePackageBaselineValidation>
     <PackageValidationBaselineVersion Condition="'$(PackageValidationBaselineVersion)' == ''">$(NetCoreAppLatestStablePackageBaselineVersion)</PackageValidationBaselineVersion>
-    <!-- PackDependsOn is the right hook in a targets file if the NuGet.Build.Tasks.Pack nuget package is used, otherwise
-         BeforePack must be used. Setting both to ensure that we are always running before other targets. --> 
-    <PackDependsOn>AddNETStandardCompatErrorFileForPackaging;IncludeAnalyzersInPackage;$(PackDependsOn)</PackDependsOn>
```

Comment:
Should we be cleaning these up elsewhere?

[https://github.com/dotnet/runtime/blob/7671b5084a41944d7a0c1fb9674e354183ec722d/eng/TraversalSdk.AfterTargets.targets#L10](https://github.com/dotnet/runtime/blob/7671b5084a41944d7a0c1fb9674e354183ec722d/eng/TraversalSdk.AfterTargets.targets#L10)

[https://github.com/dotnet/runtime/blob/57bfe474518ab5b7cfe6bf7424a79ce3af9d6657/eng/AvoidRestoreCycleOnSelfReference.targets#L8-L9](https://github.com/dotnet/runtime/blob/57bfe474518ab5b7cfe6bf7424a79ce3af9d6657/eng/AvoidRestoreCycleOnSelfReference.targets#L8-L9)